deployments/gpu_plugin/overlays/default_namespace/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+bases:
+  - ../../base
+patchesJson6902:
+- target:
+    group: apps
+    version: v1
+    kind: DaemonSet
+    name: intel-gpu-plugin
+  path: daemonset_remove_namespace.json
```

Comment:
We could just drop it from the original yaml?

Reply:
+1

Reply:
Currently all plugins (fpga, gpu, vpu) except for qat_plugin deploy their daemonset to the `kube-system` namespace by default. This is the case also in all device plugins in k8s device plugin examples. So I'd rather prefer moving qat_plugin DaemonSet to `kube-system`, too.

Reply:
the problem was that e2e testing using the k8s e2e framework was failing because it wants to set a test specific namespace...

Reply:
I see. After this patch PR #271 can use in `test/e2e/gpu/gpu.go:
```
kustomizationYaml = "deployments/gpu_plugin/overlays/no_namespace/kustomization.yaml"
```

Reply:
My feeling is that the opposite is the best approach: don't specify the namespace in the default .yamls but the user can decide:

```
$ kubectl create ns qattest
$ kubectl apply -k deployments/qat_plugin/overlay/debug -n qattest
```

whereas this is not possible right now:

```
$ kubectl apply -f deployments/gpu_plugin/gpu_plugin.yaml -n qattest
error: the namespace from the provided object "kube-system" does not match the namespace "qattest". You must pass '--namespace=kube-system' to perform this operation.
```

Reply:
Ok.

Do you think we should still provide something like `deployments/gpu_plugin/overlays/namespace_kube-system` so that there's still a way to run the plugin directly in `kube-system` like all the other plugins as well? Or would this be an unnecessary overlay as `kubectl apply -k github.../gpu_plugin -n kube-system` would do the job anyway? I don't know if other usual ways (APIs, bindings) to deploy the plugin offer configuring namespace as easily.

Ok to remove `namespace: kube-system` from the rest of the plugin yamls in later commit(s)?

Reply:
> Do you think we should still provide something like deployments/gpu_plugin/overlays/namespace_kube-system
...
> Ok to remove namespace: kube-system from the rest of the plugin yamls in later commit(s)?

+1 from my side

Reply:
I can't think of any reason not to remove the namespaces from the other yamls. As long as we....
- document that they will go in the default namespace (I presume) unless otherwise specified
- ask the other plugin/yaml owners what they think ;-)